src/components/RbzTable2.vue

```diff
@@ -705,10 +709,22 @@ const newButtonClicked = () => {
   }
 }
 
-.header-cell {
+.header-cell:not(:last-child) {
```

Comment:
It would be better to check for a more specific rule than `last-child` in this case, as the last child may change if we don't display the buttons column

```suggestion
.header-cell.column-title {
```